x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -10,6 +10,7 @@ export const alertsIndexPattern = 'logs-endpoint.alerts-*';
 export const metadataIndexPattern = 'metrics-endpoint.metadata-*';
 export const metadataCurrentIndexPattern = 'metrics-endpoint.metadata_current_*';
 export const metadataTransformPrefix = 'endpoint.metadata_current-default';
+export const metadataTransformPattern = 'endpoint.metadata_current-*';
```

Comment:
Were you able to check if the transform ever ends up as non-`default` prefix? I _think_ it might always be default, since its the superset of all namespaces. If that's the case, we can probably reuse `metadataTransformPrefix` here.

Reply:
spent a bit of time digging around but couldn't figure it out exactly so decided to just take the safer route in the interest of time. though I think you're right and that it's always `default`.